Mathlib/CategoryTheory/Sites/Coherent/ExtensiveSheaves.lean

```diff
@@ -131,4 +131,8 @@ theorem Presheaf.isSheaf_iff_preservesFiniteProducts {D : Type*} [Category D]
     rw [Presieve.isSheaf_iff_preservesFiniteProducts]
     exact ⟨inferInstance⟩
 
+noncomputable instance {D : Type*} [Category D] [FinitaryExtensive C]
```

Comment:
Can you add {D : Type*}.. in the beginning of the file (it is also used in the previous thm)? Moreover I think there are useless universes variables.